### PR TITLE
[EMB-16] Route dashboard to ember-osf-web with waffle

### PR DESCRIPTION
**Pull request by hmoco**
## Needs
Devops(?) create the switch (or, if wanted a flag or sample) `ember_dashboard` for choosing whether to use the ember or the legacy version of the dashboard (which should be functionally similar)

## Purpose
Use of ember to recreate the dashboard, rerouting to ember-osf-web when a user is logged in and the waffle switch (ember_dashboard) is active.

## Changes
Add use of waffle switch for serving dashboard.
Serve ember-osf-web when a user goes to route `/` while logged in.

## QA Notes
Test functionality of logged in dashboard, should be similar to current dashboard

## Side Effects
`/` will lead to the Ember-app

## Ticket

https://openscience.atlassian.net/browse/EMB-16

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -142,26 +143,29 @@ def index():
 
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
+        if Switch.objects.get(name='ember_dashboard').is_active():
```
Comment (sloria) [inline website/views.py]: Drive-by: you can do `waffle.switch_is_active('ember_dashboard')` here.

Comment (hmoco) [reply website/views.py]: is that better?

File: website/views.py [hunk 2]
```diff
@@ -142,26 +143,29 @@ def index():
...
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
+        if waffle.switch_is_active('ember_dashboard'):
+            return send_from_directory(ember_osf_web_dir, 'index.html')
```
Comment (icereval) [inline website/views.py]: Will this work with the ember proxied server?

File: website/views.py [hunk 3]
```diff
@@ -142,26 +143,29 @@ def index():
...
+        else:
```
Comment (binoculars) [inline website/views.py]: No need for an else here, since you're returning in the if statement. This will save the indentation/blame history.

File: website/views.py (hunk 2 above)
Comment (binoculars) [reply website/views.py]: This won't work in staging or prod. It must be proxied like in https://github.com/CenterForOpenScience/osf.io/blob/develop/website/ember_osf_web/views.py

Is that a better place for this to go?

**Comment by jamescdavis** [comment]
Closing because [PLAT-692](https://openscience.atlassian.net/browse/PLAT-692) will supersede this.
